### PR TITLE
Show queue information and add more docs links to configuration

### PR DESCRIPTION
**Pull request by JanPetr**

### CODE REVIEW

File: app/code/community/Algolia/Algoliasearch/etc/system.xml [hunk 1]
```diff
@@ -189,6 +193,18 @@
                     <show_in_website>1</show_in_website>
                     <show_in_store>1</show_in_store>
                     <expanded>0</expanded>
+                    <comment>
+                        <![CDATA[
+                        To customize autocomplete menu beyond it's configuration, you can read tutorials:
```
Comment (julienbourdeau) [inline app/code/community/Algolia/Algoliasearch/etc/system.xml]: I think it should be `its`

File: app/code/community/Algolia/Algoliasearch/etc/system.xml [hunk 2]
```diff
@@ -288,7 +304,8 @@
                             <show_in_store>1</show_in_store>
                             <comment>
                                 <![CDATA[
-                                Configure here the pages you don't want to search in.
+                                Configure here the pages you don't want to search in. <br>
+                                Pages' indexing documentation: <a target="_blank" href="https://community.algolia.com/magento/doc/m1/indexing/?utm_source=magento&utm_medium=extension&utm_campaign=magento_1&utm_term=shop-owner&utm_content=doc-link#pages-indexing">https://community.algolia.com/magento/doc/m1/indexing/#pages-indexing</a>
```
Comment (julienbourdeau) [inline app/code/community/Algolia/Algoliasearch/etc/system.xml]: I would go for singular _"page indexing"_  but I think plural form is correct too 👍

Comment (JanPetr) [inline app/code/community/Algolia/Algoliasearch/etc/system.xml]: We keep plural form everywhere - Products', Categories', Pages' ... So I'd keep it like that :)

File: app/code/community/Algolia/Algoliasearch/etc/system.xml (hunk 1 above)
Comment (JanPetr) [inline app/code/community/Algolia/Algoliasearch/etc/system.xml]: You're right! Thanks, fixed.

File: app/code/community/Algolia/Algoliasearch/etc/system.xml (hunk 2 above)
Comment (julienbourdeau) [inline app/code/community/Algolia/Algoliasearch/etc/system.xml]: Ok 👍  I didn't notice before ^^

**Pull request merged**